build-recipe-livebuild

```diff
@@ -133,7 +133,7 @@ recipe_build_livebuild() {
     # the example cited in the above bug, the metadata for sid would be incorrect
     # anyway, and we would want the ones for potato.
     if test "${DIST}" = "n/a" ; then
-	DIST=$(chroot $BUILD_ROOT su -c "sed 's/\(.*\)\/.*/\1/' /etc/debian_version")
+	DIST=$(chroot $BUILD_ROOT sh -c "sed 's/\(.*\)\/.*/\1/' /etc/debian_version")
```

Comment:
can't you simply drop the   sh -c   part?

Reply:
Removed here and in a few other places